.circleci/config.yml

```diff
@@ -92,7 +80,7 @@ jobs:
 
   clang-O3-O0:
     docker:
-      - image: multiarch/crossbuild
+      - image: ubuntu:18.10
```

Comment:
Please add clang to this bot also

Reply:
Weird.  So gcc wasn't providing a compiler?

Reply:
The bot is named gcc-valgrind though.  Why do we need clang?

Reply:
Ooh my bad.